flow-typed/teams_types.js

```diff
@@ -21,12 +21,12 @@ export type Teams$NewTeam = {
   description: string
 };
 
-
 export type Teams$Team = {
   id: number,
   name: string,
   current_team: boolean,
   role: string,
   members: number,
-  can_be_leaved: boolean
+  can_be_leaved: boolean,
```

Comment:
Sorry, grammar Nazi :). This should be written `can_be_left`. Can you rename this throughout the application?